FullStackMVCAPP/FullStackMVCAPP/DataContext/Repositories/CastleRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace FullStackMVCAPP.DataContext.Repositories
+{
+    public class CastleRepository
+    {
+        public IList<FullStackMVCAPP.Models.Castle> EntityList(GOTContext _GOTContext)
+        {
+            return _GOTContext.Castles.ToList();
+        }
```

Comment:
I suspect what you're probably after here is `IEnumerable<>` rather than `IList<>`. Some background reading [here](https://www.codeproject.com/Questions/325178/Difference-between-ilist-and-list-in-asp-net) although its a broad topic so probably best to do some self-study on.
Additionally; import reference to `FullStackMVCAPP.Models` using the `using` keyword.
`EntityList` might not be a clear name here and that something like `GetCastles()` might be more appropriate. Not suggesting you change it as naming things is hard however, when naming things also consider if the name you've chosen would make sense to someone who hasn't written this.